java-agent/src/nativeTest/kotlin/com/epam/drill/agent/configuration/ValidatorsTest.kt

```diff
@@ -0,0 +1,64 @@
+package com.epam.drill.agent.configuration
+
+import com.epam.drill.konform.validation.Invalid
+import com.epam.drill.konform.validation.Valid
+import com.epam.drill.konform.validation.Validation
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class ValidatorsTest {
+    private val hostAndPort = Validation<SomeObject> {
+        SomeObject::str required {
+            hostAndPort()
+        }
+    }
+    private val identifier = Validation<SomeObject> {
+        SomeObject::str required {
+            identifier()
+        }
+    }
+    private val validPackage = Validation<SomeObject> {
+        SomeObject::str required {
+            isValidPackage()
+        }
+    }
+
+
+    @Test
+    fun `check host and port matcher`() {
+        //valid - there are host and port
+        assertTrue { hostAndPort.validate(SomeObject("localhost:8080")) is Valid }
```

Comment:
I strongly suggest implementing separate test case for each assertion.
Otherwise first encountered assertion fail will stop the test. -> No more assertions will pass unless code is fixed -> It will mask other, possibly unrelated issues, which might pop up after the fix

The more independent _unit_ tests are the better (not the case for more end-to-end-ish tests, but these tests are definitely unit tests)